README.md

```diff
@@ -110,74 +88,50 @@ with napari.gui_qt():
+**napari** supports bidirectional communication between the viewer and the Python kernel, which is especially useful in Jupyter notebooks or from our built-in console. In this you can retrieve the locations of the points, including any additional ones you have drawn, by calling
 
 ```python
->>> points.data
+>>> viewer.layers['points'].data
 [[100, 100],
  [200, 200],
  [333, 111]]
 ```
 
-You can render and quickly browse slices of multi-dimensional arrays
-
-```python
-
-import numpy as np
-from skimage import data
-import napari
+![image](resources/screenshot-add-points-data.png)
 
-with napari.gui_qt():
-    # create fake 3d data
-    blobs = np.stack([data.binary_blobs(length=128, blob_size_fraction=0.05,
-                                        n_dim=3, volume_fraction=f)
-                     for f in np.linspace(0.05, 0.5, 10)], axis=0)
-    # add image data to the viewer
-    viewer = napari.view_image(blobs.astype(float))
-
-    # add points to the viewer
-    points = np.array(
-        [
-            [0, 0, 100, 100],
-            [0, 0, 50, 120],
-            [1, 0, 100, 40],
-            [2, 10, 110, 100],
-            [9, 8, 80, 100],
-        ]
-    )
-    viewer.add_points(
-        points, size=[0, 6, 10, 10], face_color='blue', n_dimensional=True
-    )
+You can also draw lines and polygons on an image with a `Shapes` layer. You can then adjust the properties of then shapes and vertices, including their positions and fill and stroke colors. Run `examples/add_shapes.py` to generate and interact with the following example:
 
-```
+![image](resources/screenshot-add-shapes.png)
 
-![image](resources/screenshot-nD-slicing.gif)
+You can also paint pixel-wise labels with a `Labels` layer, which is useful for creating masks for segmentation. Run `examples/add_labels.py` to generate and interact with the following example:
 
-You can draw lines and polygons on an image, including selection and adjustment of shapes and vertices, and control over fill and stroke color. Run `examples/add_shapes.py` to generate and interact with the following example.
+![image](resources/screenshot-add-labels.png)
 
-![image](resources/screenshot-add-shapes.png)
+You can render and quickly browse slices of multi-dimensional arrays in 2D or visualize them in 3D. Run `examples/nD_labels.py` to generate and interact with the following example:
 
-You can also paint pixel-wise labels, useful for creating masks for segmentation, and fill in closed regions using the paint bucket. Run `examples/add_labels.py` to generate and interact with the following example.
+![image](resources/screenshot-nD-slicing.gif)
 
-![image](resources/screenshot-add-labels.png)
+## tutorials
 
-For details checkout our [in-depth tutorials](https://napari.github.io/napari-tutorials/)
+For more details on how to use `napari` checkout our [in-depth tutorials](https://napari.github.io/napari-tutorials/)
```

Comment:
I think there should be a note here that these are a work in progress. Maybe something like:

```suggestion
For more details on how to use `napari` checkout our [in-depth tutorials](https://napari.github.io/napari-tutorials/). Please note that the tutorials are a work in progress. We are currently in the process of adding tutorials for each layer type.
```

Reply:
added